style.css

```diff
@@ -201,21 +214,22 @@ li {
   background-image: url(icons&imgs/recentworkBG.png);
   background-size: contain;
   gap: 40px;
-  width: 375px;
-  align-items: center;
+  width: var(--maxCont-width);
+  justify-items: space-between;
   align-self: center;
 }
```

Comment:
- [ ] Lastly, I would suggest you to style your footer social media container and email container so as they are aligned the way they are on the Figma design. Look at the comparison below 👇 
**Figma**
![image](https://user-images.githubusercontent.com/90222110/177845394-e8869e1b-e319-49a0-9e9a-6a1549b58d0f.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/177845572-58c7c842-fbce-4329-b254-2ee5ed3df75d.png)
Please, correct on so as to match the design you chose.🏃